public/components/AvailableTrainingVariables.vue

```diff
@@ -23,7 +23,9 @@
       :rows-per-page="numRowsPerPage"
       :summaries="availableVariableSummaries"
     >
-      <div class="available-variables-menu">
+      <div
+        class="d-flex flex-row justify-content-between align-items-center my-2 mx-1"
```

Comment:
Yes, because I noticed we were getting issues from our mixed used of custom css and bootstrap, especially when it came to predictability of how items would display in different views when it came to overall layout and scroll behaviour. The less we lean on custom classes, the most consistent the display behaviour will be, especially for reusable components.